web/src/main/java/org/cbioportal/web/DataAccessTokenController.java

```diff
@@ -67,9 +69,8 @@
     @PostConstruct
     public void postConstruct() {
         if (datMethod == null || !SUPPORTED_DAT_METHODS.contains(datMethod)) {
```

Comment:
@averyniceday since `none` is technically supported, then we can just check whether `datMethod==null` and throw some exception and if not null then always return 

```this.tokenService = this.dataAccessTokenServiceFactory.getDataAccessTokenService(this.datMethod);``` (instead of checking if it's a supported method)